ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -322,25 +325,29 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Red Circle',
           de: 'Roter Kreis',
           fr: 'Cercle rouge',
-          ko: '빨강 원형징',
+          ja: '赤 ○',
```

Comment:
Some TTS engine didn't recognize these symbols, i think it would be better just マル, 三角 and 四角.
Or add a extra `tts` entry in those trigger to make them consistent.